content/4_Gold/Modular.mdx

```diff
@@ -479,6 +479,151 @@ int main() {
 
 </CPPOnly>
 
+# Chinese Remainder Theorem
+
+## For Two Congruences
+
+<Resources>
+	<Resource
+		  source="Stanford"
+		  title="The Chinese Remainder Theorem"
+		  url="https://crypto.stanford.edu/pbc/notes/numbertheory/crt.html"
+		  starred
+	>
+		In-depth explanation
+	</Resource>
+	<Resource
+		source="cp-algo"
+		title="Chinese Remainder Theorem"
+		url="https://cp-algorithms.com/algebra/chinese-remainder-theorem.html"
+	>
+	</Resource>
+	<Resource
+		source="CMU"
+		title="Chinese Remainder Theorem"
+		url="https://www.math.cmu.edu/~mradclif/teaching/127S19/Notes/ChineseRemainderTheorem.pdf"
+	>
+	</Resource>
+</Resources>
+
+<FocusProblem problem="CRT" />
+
+### Explanation
+
+The [Chinese Remainder Theorem](https://en.wikipedia.org/wiki/Chinese_remainder_theorem) - also referred to as CRT - yields a unique
+solution to a system of simultaneous modular congruences with pairwise coprime moduli. More specifically, CRT determines a number $x$ that
+when divided by the given divisors leaves the given remainders. Mathematically, it solves the following system of modular congruences:
+
+$$
+x \equiv a_1 \pmod{m_1} \\
+x \equiv a_2 \pmod{m_2} \\
+x \equiv a_3 \pmod{m_3} \\
+\vdots\\
+x \equiv a_k \pmod{m_k} \\
+\tag{1}
+$$
+<center>$M=m_1\cdot m_2 \cdot m_3 \cdots m_k$</center>
+
+The above system has exactly one solution modulo $M$.
+
+We'll focus on the particular case for $k=2$; for two coprime moduli:
+
+$$
+x \equiv a_1 \pmod{m_1} \\
+x \equiv a_2 \pmod{m_2} \\
+\tag{2}
+$$
+<center>$M=m_1 \cdot m_2$</center>
+- Let $n_1$ and $n_2 $be the modular inverse of $m_1$ modulo $m_2$, respecively, the modular inverse of $m_2$
```

Comment:
```suggestion
- Let $n_1$ and $n_2$ be the modular inverse of $m_1$ modulo $m_2$, respectively, the modular inverse of $m_2$
```

Reply:
also respectively comes after the list, not between elements

Reply:
ok